templates/taxi/manufacturer-form.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <h1>{{ object|yesno:"Update, Create" }} Manufacturer</h1>
+  <form action = "" method="POST">
```

Comment:
The `action` attribute of the form is empty. You should set this attribute to the correct URL using Django's `{% url %}` template tag to ensure proper form submission. For example, if this form is used for creating a manufacturer, you might use `{% url 'manufacturer-create' %}`.